src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -270,9 +271,14 @@ def reproduce(self, target_path, input_path, arguments, max_time):
       max_time: Maximum allowed time for the reproduction.
 
     Returns:
-      A Result object.
+      A tuple of (return_code, output).
     """
-    raise NotImplementedError
+    runner = libfuzzer.get_runner(target_path)
+    launcher.set_sanitizer_options(target_path)
+
+    result = runner.run_single_testcase(
+        input_path, timeout=max_time, additional_args=arguments)
```

Comment:
Should we take care of the arguments in this function? I mean calling `remove_fuzzing_arguments` and adding retries like in https://github.com/google/clusterfuzz/blob/4b3e8bdc430c29768ff631ff02379b821bf4f899/src/python/bot/fuzzers/libFuzzer/launcher.py#L466

Reply:
This will be done in fuzz() instead, as we will return a crash object with the filtered args. Coming in another PR :)